src/networks.json

```diff
@@ -914,6 +914,90 @@
     "start": 536483,
     "imageIPFS": "QmeS75uS7XLR8o8uUzhLRVYPX9vMFf4DXgKxQeCzyy7vM2"
   },
+  "60001": {
+    "key": "60001",
+    "name": "Thinkium Testnet Chain 1",
+    "shortName": "ThinkiumTest1",
+    "chainId": 60001,
+    "network": "thinkiumtest1",
+    "multicall": "0xc49bc485d4f943b287edadbce45eb1a1220ffdfe",
+    "rpc": [
+      "https://test1.thinkiumrpc.net"
+    ],
+    "explorer": "https://test1.thinkiumscan.net/",
+    "start": 323327,
+    "imageIPFS": "QmY65iQYPer4P3hm5y38f7mRet4Lg43GGCbm5G6WSEB6rC"
```

Comment:
Hello @ThinkiumChain IPFS `QmY65iQYPer4P3hm5y38f7mRet4Lg43GGCbm5G6WSEB6rC` doesn't work, you should try pinning the image with a pinning service like pinnata or fleek

Reply:
I leave only one testnet , and make the IPFS image available. pls check it.